README.md

```diff
@@ -4,19 +4,31 @@ A Django based web application to manage all things IEEE NITK.
 
 ## Setup Instructions
 
-1. Run the following command to create a copy of the environment file. Change the environment variables as necessary in the copied file.
+1. Run the following command to create a copy of the environment file. Change the environment variables as necessary in
+   the copied file.
 
 ```sh
 cp env.example .env
 ```
 
-2. Run docker compose
+2. You'll need to setup the JS Toolchain, so that Tailwind and DaisyUI work properly. Install the version of `node`
```

Comment:
This is no longer relevant right? Update the docs accordingly. Mention that for Windows, this hot reloading only works with WSL. For non-WSL, I think they can either use these steps itself, or restart the container on each change.